app/assets/stylesheets/calendar.css.scss

```diff
@@ -1,107 +1,154 @@
 table#schedule {
-    width: 100%;
-    border-collapse: collapse;
-    border-style: hidden;
-    empty-cells: show;
-    font-family: sans-serif;
+  width: 100%;

```

Comment:
Properties should be ordered border-collapse, border-style, empty-cells, font-family, width